bundle/manifests/bpfman-config_v1_configmap.yaml

```diff
@@ -12,4 +12,6 @@ data:
     millisec_delay = 10000
 kind: ConfigMap
 metadata:
+  annotations: {}
+  labels: {}
```

Comment:
Why were these added?

Reply:
this change as well as TS change was side effect of `make bundle` will revert